km61/Servatmand_Mariam/laboratory1.sql

```diff
@@ -1,2 +1,129 @@
 -- LABORATORY WORK 1
 -- BY Servatmand_Mariam
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+робити вибірки з таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER Servatmand INDENTIFIED BY Servatmand
+DEFAULT TABLESPACE USERS
+TEMPORARY TABLESPACE TEMP
+
+ALTER Servatmand QUOTA 100 MB ON USERS
+ GRANT "CONNECT" TO Servatmand
+GRANT SELECT ANY TABLE TO USER
+
+
+
+
+
+
+
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Громадянин України має власне житло та автомобіль.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+  
+/* --------------------------------------------------------------------------- 
+3. Надати додаткові права користувачеві (створеному у пункті № 1) для створення таблиць, 
+внесення даних у таблиці та виконання вибірок використовуючи команду ALTER/GRANT. 
+Згенерувати базу даних використовуючи код з теки OracleScript та виконати запити: 
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+GRANT INSERT TO Servatmand
```

Comment:
Use proper statement INSERT ANY TABLE.
Use semicolon ; in the end of command.
`GRANT INSERT ANY TABLE TO Servatmand;`